cognite/client/_api/functions.py

```diff
@@ -96,6 +108,10 @@ def __init__(self, config: ClientConfig, api_version: str | None, cognite_client
         self.schedules = FunctionSchedulesAPI(config, api_version, cognite_client)
         # Variable used to guarantee all items are returned when list(limit) is None, inf or -1.
         self._LIST_LIMIT_CEILING = 10_000
+        self._CREATE_LIMIT = 1
```

Comment:
I suggest we remove `_LIST_LIMIT_CEILING` (+ from schedules API) and just add the comment above directly into the code instead. Just used once (each) anyway.